recipe/build-output.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -ex
+
+# Set a few environment variables that are not set due to
+# https://github.com/conda/conda-build/issues/3993
+export PIP_NO_BUILD_ISOLATION=False
```

Comment:
Shouldn't this be `True`? We never want build isolation.

Reply:
Guess it doesn't matter because we already have a wheel, but it's potentially confusing.

Reply:
Yeah, good point, will fix.